### ISSUE TITLE
Angled quotes in API docs

### ISSUE THREAD

**Issue by justinvp** [pulumi/docs#1510]
Angled quotes are being used in API docs. For example:

https://www.pulumi.com/docs/reference/pkg/nodejs/pulumi/kubernetes/scheduling/v1/#PriorityClass-apiVersion

<img width="590" alt="Screen Shot 2019-08-05 at 1 36 28 PM" src="https://user-images.githubusercontent.com/710598/62493596-60db5b80-b786-11e9-9fc8-69e5592f24e9.png">

We should just disable this in Hugo's config:

https://gohugo.io/getting-started/configuration/#blackfriday-options

e.g. set `smartypants` to `false` and then enable the things we want.

**Comment by justinvp** [comment]
Hmm, if we disable `smartypants`, we can't keep other things like `smartDashes` enabled. And there's no setting to specifically disable smart quotes.

**Comment by lukehoban** [comment]
> Hmm, if we disable smartypants, we can't keep other things like smartDashes enabled. And there's no setting to specifically disable smart quotes.

It's honestly more important to me that we fix this than that we get automatic smartDashes.  Should we switch our default here and then open a new nice-to-have on enabling smartDashes again in the future?

**Comment by justinvp** [comment]
> Should we switch our default here and then open a new nice-to-have on enabling smartDashes again in the future?

Yes, this is exactly the course of action I was planning to take.

**Comment by justinvp** [comment]
Found a better way to fix this that doesn't require disabling smart punctuation for the whole site. #1615
